Formula/neovim.rb

```diff
@@ -0,0 +1,58 @@
+class Neovim < Formula
+  desc "Ambitious Vim-fork focused on extensibility and agility"
+  homepage "https://neovim.io/"
+  url "https://github.com/neovim/neovim/archive/v0.2.0.tar.gz"
+  sha256 "72e263f9d23fe60403d53a52d4c95026b0be428c1b9c02b80ab55166ea3f62b5"
+  head "https://github.com/neovim/neovim.git"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
+  depends_on "jemalloc"
+  depends_on "libtermkey"
+  depends_on "libuv"
+  depends_on "libvterm"
+  depends_on "lua@5.1"
```

Comment:
I believe the reason I put this down is that it would not build with 5.2 and that I read somewhere that luajit was not 5.2 compatible which is why I changed it.